ddk-target/ddk.target

```diff
@@ -35,5 +35,9 @@
 <unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
 <repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/2.25.0/"/>
 </location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+  <repository location="https://download.eclipse.org/lsp4e/releases/0.16.0/"/>
+  <unit id="org.eclipse.lsp4e" version="0.13.3.202008291637"/>
+</location>
```

Comment:
why do we need lsp4e on the ddk target?

Reply:
It fixes an error in the c.a.t.ddk.xtext.ide project in a DDK-only workspace

Reply:
I looked at the manifest of that project, and I think adding org.eclipse.lsp4j ('j', not 'e' at the end) would be enough. Could you try that?

Reply:
done, works in my workspace